erpnext/buying/doctype/buying_settings/buying_settings.json

```
@@ -46,26 +46,26 @@
   {
    "fieldname": "po_required",
    "fieldtype": "Select",
-   "label": "Purchase Order Required for Purchase Invoice & Receipt Creation",
+   "label": "Is Purchase Order Required for Purchase Invoice & Receipt Creation?",
    "options": "No\nYes"
   },
   {
    "fieldname": "pr_required",
    "fieldtype": "Select",
-   "label": "Purchase Receipt Required for Purchase Invoice Creation",
+   "label": "Is Purchase Receipt Required for Purchase Invoice Creation?",
    "options": "No\nYes"
   },
   {
    "default": "0",
    "fieldname": "maintain_same_rate",
    "fieldtype": "Check",
-   "label": "Maintain same rate throughout purchase cycle"
+   "label": "Maintain Same Rate Throughout the Purchase Cycle"
   },
   {
    "default": "0",
    "fieldname": "allow_multiple_items",
    "fieldtype": "Check",
-   "label": "Allow Item to be added multiple times in a transaction"
+   "label": "Allow Item to Be Added Multiple Times in a Transaction"
```

```suggestion
   "label": "Allow Item To Be Added Multiple Times in a Transaction"
```
Refer **Capitalize Verbs and Helping Verbs** here:
https://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html